src/benchmarklib/synthetic_table_generator.cpp

```diff
@@ -149,6 +150,19 @@ std::shared_ptr<Table> SyntheticTableGenerator::generate_table(
             }
           }
 
+          std::vector<bool> null_values;
+
+          if (null_ratio.has_value()) {
```

Comment:
Add documentation on how we distribute null values here.

Reply:
Resolved in e8b8bb797c3adbc7ed020a820dd14d294087dcc1